src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -184,7 +184,12 @@ def _check_if_all_defined_integrations_exists(
 ):
     existing_integrations = {
         i["name"].lower()
-        for i in om.show(object_type="integration", cursor_class=DictCursor, like=None)
+        for i in om.show(
+            object_type="integration",
+            cursor_class=DictCursor,
+            like=None,
+            scope=(None, None),
```

Comment:
Is this change necessary if we use default value?

Reply:
good point, will remove unnecessary change